python/binaryview.py

```diff
@@ -3179,20 +3179,18 @@ def get_next_data_after(self, addr):
 
 	def get_next_data_var_after(self, addr):
 		"""
-		``get_next_data_var_after`` retrieves the next virtual address of the next :py:Class:`DataVariable`
+		``get_next_data_var_after`` retrieves the next :py:Class:`DataVariable`, or None.
 
 		:param int addr: the virtual address to start looking from.
-		:return: the virtual address of the next :py:Class:`DataVariable`
-		:rtype: int
+		:return: the next :py:Class:`DataVariable`
+		:rtype: DataVariable
 		:Example:
 
-			>>> hex(bv.get_next_data_var_after(0x10000000))
-			'0x1000003cL'
-			>>> bv.get_data_var_at(0x1000003c)
+			>>> bv.get_next_data_var_after(0x10000000)
 			<var 0x1000003c: int32_t>
 			>>>
 		"""
-		return core.BNGetNextDataVariableAfterAddress(self.handle, addr)
+		return self.data_vars.get(core.BNGetNextDataVariableAfterAddress(self.handle, addr), None)
```

Comment:
```suggestion
		return self.data_vars.get(core.BNGetNextDataVariableAfterAddress(self.handle, addr))
```
You don't need the `, None` because it's the default behavior of `dict.get`

Reply:
Thanks!